modules/mobile/app/controllers/mobile/v0/military_information_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require_dependency 'mobile/application_controller'
+
+module Mobile
+  module V0
+    class MilitaryInformationController < ApplicationController
```

Comment:
Is there an API doc PR that goes along with this new endpoint? If so, could you link in the PR body please?

Reply:
Added to the PR